src/components/frontend.jsx

```diff
@@ -1,63 +1,56 @@
-                <a href="/vuejs" className='sm:mb-10 pl-2 pr-2 flex flex-col shadow-lg shadow-[#00df9a] transition-shadow  hover:bg-opacity-90  duration-[320ms] bg-gray-900 rounded-lg text-green-400 hover:bg-[#00df9a] hover:shadow-none transition-color hover:text-white transition-color'>
-                    <h2 className='text-center font-bold text-3xl px-4 py-6'>VUE JS</h2>
-                </a>
-                <img src={arrow} className="w-0 h-0 max-w-screen-xl sm:w-20 sm:h-20"/>
-                <img src={downarrow} className="w-20 h-20 max-w-screen-xl sm:w-0 sm:h-0"/>
-                <a href="/threejs" className='sm:mb-10 pl-2 pr-2 flex flex-col shadow-lg shadow-[#00df9a] transition-shadow  hover:bg-opacity-90  duration-[320ms] bg-gray-900 rounded-lg text-green-400 hover:bg-[#00df9a] hover:shadow-none transition-color hover:text-white transition-color'>
-                    <h2 className='text-center font-bold text-3xl px-4 py-6'>THREE JS</h2>
-                </a>
-                <img src={arrow} className="w-0 h-0 max-w-screen-xl sm:w-20 sm:h-20"/>
-                <img src={downarrow} className="w-20 h-20 max-w-screen-xl sm:w-0 sm:h-0"/>
-                <a href="/reactnative" className='sm:mb-10 pl-2 pr-2 flex flex-col shadow-lg shadow-[#00df9a] transition-shadow  hover:bg-opacity-90  duration-[320ms] bg-gray-900 rounded-lg text-green-400 hover:bg-[#00df9a] hover:shadow-none transition-color hover:text-white transition-color'>
-                    <h2 className='text-center font-bold text-3xl px-4 py-6'>REACT NATIVE</h2>
-                </a>
-                <img src={arrow} className="w-0 h-0 max-w-screen-xl sm:w-20 sm:h-20"/>
-                <img src={downarrow} className="w-20 h-20 max-w-screen-xl sm:w-0 sm:h-0"/>
-                <a href="/electron" className='sm:mb-10 pl-2 pr-2 flex flex-col shadow-lg shadow-[#00df9a] transition-shadow  hover:bg-opacity-90  duration-[320ms] bg-gray-900 rounded-lg text-green-400 hover:bg-[#00df9a] hover:shadow-none transition-color hover:text-white transition-color'>
-                    <h2 className='text-center font-bold text-3xl px-4 py-6'>ELECTRON</h2>
-                </a>
+  const courses = [
+    { name: "HTML / CSS", link: "/html_css" },
+    { name: "TAILWIND", link: "/tailwind" },
+    { name: "JAVA SCRIPT", link: "/javascript" },
+    { name: "REACT JS", link: "/reactjs" },
+    { name: "NEXT JS", link: "/nextjs" },
+    { name: "VUE JS", link: "/vuejs" },
+    { name: "THREE JS", link: "/threejs" },
+    { name: "REACT NATIVE", link: "/reactnative" },
+    { name: "ELECTRON", link: "/electron" },
+  ];
 
-            </div>
+  return (
+    <div className="bg-gray-800 min-h-screen py-20">
+      <h1 className="w-full mb-24 text-white px-4 tracking-tight text-center text-5xl">
+        Complete Frontend Roadmap
+      </h1>
 
-        </div>
-    );
+      <div className="flex flex-wrap justify-center gap-4 mx-4 md:mx-20">

```

Comment:
You can use `grid-cols-7` which allows for better control over the placement of the courses and arrows.